README.md

```diff
@@ -505,6 +506,94 @@ The supported `PlayerView` events for subtitles are:
 
 You might check out a complete subtitle example in the [`example/`](https://github.com/bitmovin/bitmovin-player-react-native/tree/development/example) app.
 
+### Setting up ads
+
+The Bitmovin Player SDKs are capable of serving Ads out of the box and there are two ways they can be
```

Comment:
The term "serving" could be misleading, as in ads terms an Ad Server is a totally different concept.

```suggestion
The Bitmovin Player SDKs are capable of displaying Ads out of the box and there are two ways they can be
```

Reply:
Done 08389af9709478fde072f507096ce7670b345533